ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDataExtensions.kt

```diff
@@ -48,3 +49,10 @@ val PaywallData.Configuration.Colors.accent1Color: Color
     get() = accent1?.let { Color(it.colorInt) } ?: callToActionForegroundColor
 val PaywallData.Configuration.Colors.accent2Color: Color
     get() = accent2?.let { Color(it.colorInt) } ?: accent1Color
+
+val PaywallData.iconUrlString: String
```

Comment:
This lives in `PaywallData`, which might not be the best place... But moving it inside `config.images` was tricky, since it depends on data on the `PaywallData`, so I thought this was ok. We could also create a wrapper to contain all these once we use more images.

Reply:
Right, we should probably work on that transformation soon too, since it will help when we start parsing variables and a bunch of other stuff.